src/rules/flake8_bugbear/rules/getattr_with_constant.rs

```diff
@@ -47,7 +51,10 @@ pub fn getattr_with_constant(checker: &mut Checker, expr: &Expr, func: &Expr, ar
 
     let mut diagnostic =
         Diagnostic::new(violations::GetAttrWithConstant, Range::from_located(expr));
-    if checker.patch(diagnostic.kind.rule()) {
+
+    // Don't autofix if name would be mangled by Python's "private variables"
+    // handling
+    if checker.patch(diagnostic.kind.rule()) && !is_mangled_private(value.as_str()) {
```

Comment:
I think we should put this just after line 43, and avoid raising this error. What do you think?

(We need to make the same change in `setattr_with_constant.rs` too.)

Reply:
Made the change for setattr, although I think it makes sense to still raise the error - this matches the upstream check's behavior.

Reply:
Is it intentional, though, that bugbear raises on these cases?

Reply:
I don't see anything in bugbear's issues or code to indicate it's intentional - I can make that change if you'd prefer that.

Reply:
I think I'd prefer it, if you don't mind. It just strikes me as very unlikely to be a false negative, and as-is, it'll probably cause confusion.

Reply:
Sounds good! Updated with the change.